source/PlayState.hx

```diff
@@ -1263,7 +1263,7 @@ class PlayState extends MusicBeatState
 					if(gf != null) gf.playAnim('scared', true);
 					boyfriend.playAnim('scared', true);
 
-				case "winter-horrorland":
+				/*case "winter-horrorland":
```

Comment:
Winter Horrorland won't have the desired effect since the code is commented out without a proper replacement.

Reply:
I’ll do it it soon